### PR TITLE
Feedback loop submit button

### PR DESCRIPTION
**Pull request by samhstn**
Adds submit button to each feedback loop box

Ref #179

### CODE REVIEW

File: resources/templates/resources/single_visited.html [hunk 1]
```diff
@@ -38,6 +38,7 @@ <h4 class="mt0 mb1 f4">{{v.heading}}</h4>
           <input type="hidden" name="resource_title" value="{{v.title}}" />
           <input type="hidden" name="liked" value="{{v.liked_value}}" />
           <input type="hidden" name="resource_name" value="{{v.heading}}" />
+          {% include "resources/primary-button.html" with value="Submit" class="mv3 fr" %}
         </form>
```
Comment (Danwhy) [inline resources/templates/resources/single_visited.html]: This should be inline with the search bar

**Comment by samhstn** [comment]
So just to clarrify, it is currently like this:
screen shot 2017-08-24 at 13 09 33

But we want it like this:

screen shot 2017-08-24 at 13 14 54

**Comment by Danwhy** [comment]
:+1: Looks great

**Review by Danwhy — APPROVED** [review]

**Review by SimonLab — APPROVED** [review]
:+1:

**Pull request merged**